gateway-manager/src/manage_confluent_cloud.py

```diff
@@ -0,0 +1,605 @@
+    logout()
+
+
+def create_tenant(realm):
+    _connect()
+    account = get_or_create_tenant_sa(realm)
+    allowed_resource = f'{realm}.'
+    for operation in ALL_TENANT_PERMISSION:
+        for resource_type, resource_id, wildcard in [
+                ('topic', allowed_resource, True),
+                ('consumer-group', allowed_resource, True)
+        ]:
+            acl_create(
+                account.id,
+                resource_id,
+                resource_type=resource_type,
+                operation=operation,
+                extended_acl=wildcard,
+                cluster=False
+            )
+    acl_list(service_account_id=account.id)
+    LOGGER.info('Regular tenant "{realm}" created.')
+    logout()
+
+
+def create_key(account_name, desc=None):
+    _connect()
+    if not desc:
+        now = str(datetime.now().isoformat())
+        desc = f'Account:{account_name} generated: {now}'
+    LOGGER.info(f'Creating key for account: {account_name}')
+    account = _get_service_account_by_name(account_name)
+    key = api_key_create(account.id, desc)
+    LOGGER.info('~~~ SAVE THIS KEY!!! IT CANNOT BE RETRIEVED ~~~')
+    LOGGER.info(key)
+    logout()
+
+
+def list_accounts():
+    _connect()
+    accounts = get_service_accounts()
+    for account in accounts:
+        LOGGER.info(account)
+    logout()
+
+
+def list_acls(account_name=None):
+    _connect()
+    sa_id = None
+    if account_name:
+        account = _get_service_account_by_name(account_name)
+        sa_id = account.id
+        acls = acl_list(service_account_id=sa_id)
+        LOGGER.info(account)
+        for acl in acls:
+            LOGGER.info(acl)
+    else:
+        accounts = get_service_accounts()
+        acls = acl_list(service_account_id=sa_id)
+        for account in accounts:
+            LOGGER.info(account)
+            for acl in acls:
+                if account.id == acl.id:
+                    LOGGER.info(acl)
+    logout()
+
+
+def list_api_keys():
+    _connect()
+    keys = api_key_list()
+    for key in keys:
+        LOGGER.info(key)
+    logout()
+
+
+if __name__ == '__main__':
+    COMMANDS = {
+        'ADD_SUPERUSER': create_superuser,
+        'GRANT_SUPERUSER': grant_superuser,
+        'ADD_TENANT': create_tenant,
+        'CREATE_KEY': create_key,
+        'LIST_SERVICE_ACCOUNTS': list_accounts,
+        'LIST_ACLS': list_acls,
+        'LIST_API_KEYS': list_api_keys
+    }
+
+    command = sys.argv[1]
+    if command.upper() not in COMMANDS.keys():
+        LOGGER.critical(f'No command: {command}')
+        sys.exit(1)
+    requirements = [
+        CC_CLI_PATH,
+        CC_API_USER,
+        CC_API_PASSWORD,
+        CC_CLUSTER_NAME,
+        CC_ENVIRONMENT_NAME
+    ]
+    if not all(requirements):
+        LOGGER.error(f'CCloud environment variables are missing.')
+        sys.exit(1)
+    try:
+        fn = COMMANDS[command]
+        args = sys.argv[2:]
+        fn(*args)
+    except Exception as e:
```

Comment:
I find useful to include the _connect/logout calls in the try/except clause better than within the command functions. To make sure that you don't leave open connections... :thinking: 

```python
try:
    _connect()
    # ...
except Exception as e:
    # ...
finally:
    try:
        logout()
    except Exception:
        pass  # it does not matter
```